### DISCUSSION TITLE
Changes for Upstream Avalonia to Better Align with WinUI

### DISCUSSION THREAD

**Discussion by robloo** [amwx/FluentAvalonia#226] [General]
@amwx, I'm going through trying to find the critical missing functionality Avalonia needs for those of us coming over from WPF/UWP/WinUI and trying to get that in before the 11.0 release. I've been adding things like Rounded Rectagle, Loaded/Unloaded/SizeChanged events, etc. as you know.

Is there any missing functionality you recall that would have been nice to have when you first created this repo and ported over from the WinUI themes? I know you've already added a lot of things yourself. However, if you can think of anything specific please let me know and if it it aligns with a future port on my end I'll strongly consider adding it upstream.

**Comment by amwx** [comment]
Off the top of my head I can't think of anything else I'd classify as "critical missing functionality" like those things you've already done. I think those are the big ones that have always required a workaround in porting controls.

**Reply by robloo** [reply]
Ok, thanks! If something comes up -- event non-critical but annoying/problematic -- just let me know. You have looked at WinUI code compared to Avalonia a lot more than others I suspect.
